e2e/_suites/fleet/features/stand_alone_agent.feature

```diff
@@ -54,7 +54,7 @@ Examples: Ubi8
 @run_fleet_server
 Scenario Outline: Deploying a <image> stand-alone agent with fleet server mode
   When a "<image>" stand-alone agent is deployed with fleet server mode
-  Then the agent is listed in Fleet as "online"
+  Then the agent is listed in the Fleet UI as "online"
```

Comment:
I think this will do the trick, although we'd like to read that this step belongs to the standalone agent in the spec. At least until we have a better abstraction for both existing steps (this one and the one for the managed mode) at the test suite struct level (maybe merging both structs into one instead of decoupling behaviours, at the end of the day, the end results should be the same: operate on an agent)?

Wdyt about this?
```suggestion
  Then the standalone agent is listed in Fleet as "online"
```